### PR TITLE
Fix tekton chains deploy

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**

### CODE REVIEW

File: developer/config.yaml [hunk 1]
```diff
@@ -6,10 +6,10 @@
 cluster_type: openshift
 
 # git_url refers to a git repo to be considered as the source of truth for Argo CD applications.
-git_url: https://github.com/openshift-pipelines/pipeline-service.git
+git_url: https://github.com/AndrienkoAleksandr/pipeline-service.git
```
Comment (bnallapeta) [inline developer/config.yaml:9]: Please revert this.

File: developer/openshift/gitops/argocd/pipeline-service.yaml [hunk 2]
```diff
@@ -10,8 +10,8 @@ spec:
     server: https://kubernetes.default.svc
   source:
     path: developer/openshift/gitops/argocd/pipeline-service
-    repoURL: https://github.com/openshift-pipelines/pipeline-service.git
-    targetRevision: main
+    repoURL: https://github.com/AndrienkoAleksandr/pipeline-service.git
+    targetRevision: fix-tekton-chains
```
Comment (bnallapeta) [inline developer/openshift/gitops/argocd/pipeline-service.yaml:14]: Please revert this.

**Review by bnallapeta — APPROVED** [review]
Looks good once the git reference is updated and commits are squashed.

File: developer/config.yaml (hunk 1 above)
Comment (AndrienkoAleksandr) [reply developer/config.yaml]: Done

File: developer/openshift/gitops/argocd/pipeline-service.yaml (hunk 2 above)
Comment (AndrienkoAleksandr) [reply developer/openshift/gitops/argocd/pipeline-service.yaml]: Done.

**Pull request merged**